src/renderer/components/monaco-editor/monaco-editor.tsx

```diff
@@ -288,14 +292,23 @@ class NonInjectedMonacoEditor extends React.Component<MonacoEditorProps & Depend
   // avoid excessive validations during typing
   validateLazy = debounce(this.validate, 250);
 
+  get initialHeight() {
+    return this.props.getEditorHeightFromLinesCount(this.model.getLineCount());
+  }
+
   render() {
-    const { className, style } = this.props;
+    const { className, style, setInitialHeight } = this.props;
+
+    const css: React.CSSProperties = {
+      ...style,
+      height: setInitialHeight ? this.initialHeight : style?.height,
```

Comment:
Shouldn't own `style?.height` have more priority over some initial height? E.g.

```suggestion
      height: style?.height ?? this.initialHeight,
```